wagtail_storages/tests/settings.py

```diff
@@ -15,7 +17,7 @@
     "wagtail.admin",
     "wagtail.documents",
     "wagtail.users",
-    "wagtail.core",
+    "wagtail" if WAGTAIL_VERSION >= (3, 0) else "wagtail.core",
```

Comment:
Thats interesting I don't think it's required but the new project template for Wagtail 3.0 does use `wagtail`

Reply:
I think `wagtail updatemodulepaths` auto renamed it.